Makefile

```diff
@@ -95,11 +95,17 @@ julia-ui-release julia-ui-debug : julia-ui-% : julia-src-%
 julia-inference : julia-base julia-ui-$(JULIA_BUILD_MODE) $(build_prefix)/.examples
 	@$(MAKE) $(QUIET_MAKE) -C $(BUILDROOT) $(build_private_libdir)/inference.ji JULIA_BUILD_MODE=$(JULIA_BUILD_MODE)
 
+ifneq ($(CPUID_SPECIFIC_BINARIES), 0)
+CPUID_TAG = _$(shell $(JULIA_EXECUTABLE) --cpuid)
```

Comment:
needs `$(call spawn`

and should have a comment that this is being lazy evaluated (rather than using `:=`)